src/app/components/home/home-top-banner.component.scss

```diff
@@ -0,0 +1,74 @@
+@import '../../styles/colors';
+@import '../../styles/dimensions';
+@import '../../styles/typography';
+
+.home-header-container {
+  display: flex;
+  background: linear-gradient($color-cerise-600, $color-primary-cerise);
+  color: $white;
+  flex-flow: row-reverse;
+  @media screen and (max-width: $md-device-width) {
+    flex-flow: column;
+  }
+
+  .home-header-map-image {
+    width: 426px;
+    padding-right: 0;
+    @media screen and (max-width: $md-device-width) {
+      margin: 0 auto;
+    }
+    @media screen and (max-width: 426px) {
+      width: 100%;
+    }
+  }
+
+  .home-header-info-container {
+    padding: 40px;
```

Comment:
Properties should be ordered width, padding, text-align

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)